core-web/libs/block-editor/src/lib/extensions/image-tabview-form/plugins/bubble-image-tabview-form.plugin.ts

```diff
@@ -0,0 +1,207 @@
+import { ComponentRef } from '@angular/core';
+import { Subject } from 'rxjs';
+
+import { Editor, posToDOMRect } from '@tiptap/core';
+import { EditorState, Plugin, PluginKey, Transaction } from 'prosemirror-state';
+import { EditorView } from 'prosemirror-view';
+import tippy, { Instance, Props } from 'tippy.js';
+
+import { ImageNode } from '@dotcms/block-editor';
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
+
+import { ImageTabviewFormComponent } from '../image-tabview-form.component';
+
+interface PluginState {
+    open: boolean;
+}
+
+export interface BubbleLinkFormProps {
+    pluginKey: PluginKey;
+    editor: Editor;
+    element: HTMLElement;
+    tippyOptions?: Partial<Props>;
+    component?: ComponentRef<ImageTabviewFormComponent>;
+}
+
+export type BubbleImageTabFormViewProps = BubbleLinkFormProps & {
+    view: EditorView;
+};
+
+export class BubbleImageTabFormView {
+    public editor: Editor;
+
+    public element: HTMLElement;
+
+    public view: EditorView;
+
+    public tippy: Instance | undefined;
+
+    public hola = false;
+
+    public tippyOptions?: Partial<Props>;
+
+    public pluginKey: PluginKey;
+
+    public component?: ComponentRef<ImageTabviewFormComponent>;
+
+    private $destroy = new Subject<boolean>();
+
+    constructor({
+        editor,
+        element,
+        view,
+        tippyOptions = {},
+        pluginKey,
+        component
+    }: BubbleImageTabFormViewProps) {
+        this.editor = editor;
+        this.element = element;
+        this.view = view;
+
+        this.tippyOptions = tippyOptions;
+
+        // Detaches menu content from its current parent
+        this.element.remove();
+        this.element.style.visibility = 'visible';
+        this.pluginKey = pluginKey;
+        this.component = component;
+        this.component.instance.languageId = this.editor.storage.dotConfig.lang;
+
+        this.component.instance.selectedContentlet.subscribe((contentlet) => {
```

Comment:
unsubscribe ?

Reply:
Or maybe emit an output no?

Reply:
Or pass a callback to the component.

Reply:
It is a `@Output` already but a `callback` sounds good!

https://github.com/dotCMS/core/blob/b037430d86b3488ac188c44b8a75cc9892e37259/core-web/libs/block-editor/src/lib/extensions/image-tabview-form/components/dot-image-card-list/dot-image-card-list.component.ts#L17